CHANGELOG.md

```diff
@@ -155,6 +155,8 @@ Adding a new version? You'll need three changes:
   This was caused by a bug in `NodeAgent` that was sending the updates
   despite the fact that the configuration status was not changed.
   [#4324](https://github.com/Kong/kubernetes-ingress-controller/pull/4324)
+- Properly construct targets for IPv6-only clusters.
```

Comment:
I don't think this will come to the already-released 2.10.3. I think it should go to the `Unreleased` section, or to `2.10.4` section if we backport it to 2.10.

Reply:
👍 for using `Unreleased` here. 2.10.4 in the backport.